libfwupdplugin/fu-dpaux-device.c

```diff
@@ -172,6 +175,47 @@ fu_dpaux_device_setup(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+/**
+ * fu_dpaux_device_get_offset:
+ * @self: a #FuDpauxDevice
+ *
+ * Gets the DPCD offset.
+ *
+ * Returns: integer
+ *
+ * Since: 2.0.8
+ **/
+goffset
+fu_dpaux_device_get_dpcd_offset(FuDpauxDevice *self)
+{
+	FuDpauxDevicePrivate *priv = GET_PRIVATE(self);
+	g_return_val_if_fail(FU_IS_DPAUX_DEVICE(self), G_MAXOFFSET);
+	return priv->dpcd_offset;
+}
+
+/**
+ * fu_dpaux_device_set_dpcd_offset:
+ * @self: a #FuDpauxDevice
+ * @dpcd_offset: integer
+ *
+ * Sets the DPCD offset. Must be either FU_DPAUX_DEVICE_DPCD_OFFSET_SOURCE_DEVICE,
+ * FU_DPAUX_DEVICE_DPCD_OFFSET_SINK_DEVICE, or FU_DPAUX_DEVICE_DPCD_OFFSET_BRANCH_DEVICE.
```

Comment:
the constants need a % prefix to make them hyperlinks in the docs -- e.g. `%FU_DPAUX_DEVICE_DPCD_OFFSET_SOURCE_DEVICE`